src/Widgets/PaymentPlans/PaymentPlans.module.css

```diff
@@ -47,6 +47,9 @@
 .plan.active {
   border-bottom: 3px solid var(--blue-300);
 }
+.plan.active.polychrome {
+  border-bottom: 3px solid var(--red-600);
```

Comment:
You can just add a border-color as the rest of the property is set elsewhere and we only want to design the color.